tfjs-layers/src/engine/container.ts

```diff
@@ -972,27 +972,27 @@ export abstract class Container extends Layer {
    *    subclasses: ['LayersModel']
    * }
    */
-  getLayer(name?: string, index?: number): Layer {
-    if (index != null) {
-      if (this.layers.length <= index) {
+  getLayer(input?: string|number): Layer {
```

Comment:
This might cause breaking change for `getLayer(null, 1)` usage.

We could make it like `getLayer(nameOrIndex: string|number, index?: number)`, similar as https://github.com/tensorflow/tfjs/blob/master/tfjs-core/src/engine.ts#L441

Reply:
Done! Thanks!